app/models/ability.rb

```diff
@@ -196,15 +188,13 @@ def signed_in_with_cfp_role(user)
                          commentable_id: Event.where(program_id: Program.where(conference_id: conf_ids_for_cfp).pluck(:id)).pluck(:id)
 
     # Abilities for Role (Conference resource)
-    can :index, Role
-
-    can :manage, Role do |role|
+    can :read, Role do |role|
```

Comment:
Even though what you are doing is technically correct, I prefer not using read but explicit actions instead. 
* It will make the code more readable (not everyone knows what read corresponds to)
* And it will protect us, in the future, from changes made to read in cancancan gem, which we might miss when updating it